content/subscription/build-cloud/manage-seats.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Manage seats and invites
+description: Learn how to add and remove seats for your Docker Build Cloud subscription, and invite members to your team.
+keywords: subscription, team, business, features, build, cloud, Build Cloud, remote builder, manage users, add seats, remove seats
+---
+
+You can add seats and manage invitations to your Docker Build Cloud Team in the Docker Build Cloud dashboard.
+
+> **Note**
+>
+>If you have a Docker Build Cloud Business subscription, you can add and remove seats by working with your account executive, then assign your purchased seats in the Docker Build Cloud dashboard.
+
+## Add seats
+
+1. Sign in to your [Docker account](https://login.docker.com/) and go to [Docker Build Cloud](https://build.docker.com/).
+2. Select **Settings & billing**.
+3. Under your current plan, select **Add seats**.
+4. Select the number of seats you want to add, then select **Add seats**.
+
+The number of seats will be charged to your payment information on file, and are added immediately.
+
+## Remove seats
+
+1. Sign in to your [Docker account](https:/hub.docker.com/login) and go to [Docker Build Cloud](https://build.docker.com/).
+2. Select **Settings & billing**.
+3. Under your current plan, select **Remove seats**.
+4. Enter the total number of seats you want to keep on your plan, then select **Remove seats**.
+
+The charge for the reduced seat count will be reflected on the next billing cycle. Optionally, you can cancel the seat downgrade any time before the next billing cycle.
+
+## Invite users
+
+As an owner of the Docker Build Cloud team, you can invite members to access cloud builders. To invite team members to your team in Docker Build Cloud:
+
+1. Sign in to your [Docker account](https:/hub.docker.com/login) and go to [Docker Build Cloud](https://build.docker.com/).
+2. Select **User management**.
+3. Select **Invite**.
+4. Enter the email address or Docker ID of the member you want to invite. If the invitee doesn’t have a Docker ID, you need to enter their email address so they can create an account when they accept the invite.
```

Comment:
```suggestion
4. Enter the email address or Docker ID of the member you want to invite. If the invitee doesn’t have a Docker ID, enter their email address so they can create an account when they accept the invite.
```

Reply:
if the user has a Docker ID, can you just enter the email anyway? Or do you have to enter it differently depending on if they already have an account or not?

Reply:
Yeah, you can do email either way but if there's no Docker ID it has to be email. I can tweak this wording!